modules/developer-cli-odo-adding-storage-to-the-application-components.adoc

```diff
@@ -5,15 +5,67 @@
 [id="adding-storage-to-the-application-components_{context}"]
 = Adding storage to the application components
 
-Persistent storage keeps data available between restarts of {odo-title}. You can add storage to your components with the `odo storage` command.
+Persistent storage keeps data available between restarts of {odo-title}.
+Use the `odo storage` command to add persistent data to your application. Examples of data that must persist include database files, dependencies, and build artifacts, such as a `.m2` Maven directory.
 
 .Procedure
 
-* Add storage to your components:
+. Add the storage to your component:
 +
 [source,terminal]
 ----
-$ odo storage create nodestorage --path=/opt/app-root/src/storage/ --size=1Gi
+$ odo storage create <storage-name> --path=<path-to-the-directory> --size=<size>
```

Comment:
s/<storage-name>/<storage_name>/
s/<path-to-the-directory>/<path_to_the_directory>/

Reply:
@bobfuru my apologies! Somehow I missed the notification about your comment. 
Fixed.

Reply:
@boczkowska - LGTM, let me know if you're ready for merge/CP

Reply:
@bobfuru it is ready to be merged and cherry-picked. Could you please do it? I don\t have the merging rights.
Thank you.